addons/website_hr_recruitment/views/hr_recruitment_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_hr_recruitment_tree_url" model="ir.ui.view" >
+        <field name="name">hr.recruitment.tree.inherit.url</field>
+        <field name="model">hr.recruitment.source</field>
+        <field name="inherit_id" ref="hr_recruitment.hr_recruitment_source_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='create_alias']" position="after">
```

Comment:
Why is this a better position than the previous one ? If there is a good reason then I'm all for it, but I was wondering... :)